src/load_data.jl

```diff
@@ -1,150 +1,28 @@
-using HDF5, JSON, Compat, YAML, CSV
+using JSON, Compat, CSV, Mmap
 
 """
     load_data(filename)
 
-Load hdf5 header and data
+Load data file
 """
-function load_data(filename::AbstractString)
```

Comment:
Maybe we shouldn't get rid of the old auspex load entirely?  Could make loading old data a pain.

Reply:
that's a good point, I'l bring it back as a dispatch